.github/actions/monitor-cdk-verified-batches/batch_verification_monitor.sh

```diff
@@ -1,18 +1,30 @@
 #!/bin/bash
 
 # This script monitors the verification progress of zkEVM batches.
-# Usage: ./batch_verification_monitor <verified_batches_target> <timeout>
+
+# Check if the required arguments are provided.
+if [ "$#" -lt 2 ]; then
+  echo "Usage: $0 <verified_batches_target> <timeout> [rpc_service]"
+  exit 1
+fi
```

Comment:
I prefer to use `[[ ... ]]` but not a big deal:
https://google.github.io/styleguide/shellguide.html#s6.3-tests

Reply:
Thanks for the suggestion. I changed it to `[[ ... ]]`.